docassemble/AssemblyLine/al_general.py

```diff
@@ -259,7 +260,50 @@ def gender_fields(self, show_help=False):
         },
         self_described_input
       ]
-              
+      
+  @property
+  def gender_male(self):
+    """Provide True/False for 'male' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'male'
+  
+  @property
+  def gender_female(self):
+    """Provide True/False for 'female' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'female'
+  
+  @property
+  def gender_other(self):
+    """Provide True/False for 'other' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on for forms without more inclusive options."""
+    return ((self.gender != 'male') and (self.gender != 'female'))
+  
+  @property
+  def gender_nonbinary(self):
+    """Provide True/False for 'nonbinary' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    
+    return self.gender == 'nonbinary'
+  
+  @property
+  def gender_unknown(self):
+    """Provide True/False for 'unknown' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'unknown'
+  
+  @property
+  def gender_undisclosed(self):
+    """Provide True/False for 'prefer-not-to-say' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""
+    return self.gender == 'prefer-not-to-say'  
+  
+  @property
+  def gender_self_described(self):
+    """Provide True/False for 'self-described' gender to assist with checkbox filling
+    in PDFs with "skip undefined" turned on."""    
+    return self.gender == 'self-described'    
```

Comment:
Unless I'm mistaken, line 242 changes the user's gender to be whatever they type in, in the self-described box. So this should probably be:
```suggestion
    return self.gender not in ['male', 'female', 'nonbinary', 'unknown', 'prefer-not-to-say']
```